ros_buildfarm/templates/snippet/builder_system-groovy_check-for-failing-jobs.xml.em

```diff
@@ -0,0 +1,82 @@
+@(SNIPPET(
+    'builder_system-groovy',
+    command=
+"""// CHECK FOR FAILING JOBS
+import hudson.model.Job
+
+// Control the number of failures in a row to check
+min_number_of_failures = 5
+// Control the minimun amount of time failing
+min_days_failing = 7
+// rosdistro to search
+rosdistro = @ros_distribution
```

Comment:
arg! I changed the name just before the commit. I set it to  `ros_distro` but I can not make it to expand playing with the quotes and the @. Any idea?